bokehjs/src/lib/core/has_props.ts

```diff
@@ -333,7 +333,6 @@ export abstract class HasProps extends Signalable() implements Equals, Printable
   private _setv(changes: Map<Property, unknown>, options: HasProps.SetOptions): void {
```

Comment:
Should we renamed this to `_setv_nonsilent` or something just to make it clear?

Reply:
I'm going to replace `setv()` and `_setv()` completely at some point (hopefully even in 2.2 cycle), so let's take a pause on renaming this.